### ISSUE TITLE
Zynq drivers break hw_zynq build

### ISSUE THREAD

**Issue by chelmuth** [genodelabs/genode-world#37]
```
  Program drivers/sd_card/spec/zynq/sd_card_drv
    COMPILE  main.o
/home/krishna/tmp/genode-world.git/src/drivers/sd_card/spec/zynq/main.cc:20:20: fatal error: driver.h: No such file or directory
 #include <driver.h>
                    ^
compilation terminated.
make[2]: *** [main.o] Error 1
make[2]: Target `all' not remade because of errors.
make[1]: *** [sd_card_drv.prg] Error 2
  Library config
  Program drivers/i2c/spec/zynq/zynq_i2c_drv
    COMPILE  main.o
In file included from /home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/main.cc:25:0:
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:90:3: error: ‘I2C0_MMIO_BASE’ is not a member of ‘Genode::Board_base’
  {Genode::Board_base::I2C0_MMIO_BASE, Genode::Board_base::I2C_MMIO_SIZE},
   ^
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:90:39: error: ‘I2C_MMIO_SIZE’ is not a member of ‘Genode::Board_base’
  {Genode::Board_base::I2C0_MMIO_BASE, Genode::Board_base::I2C_MMIO_SIZE},
                                       ^
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:91:3: error: ‘I2C1_MMIO_BASE’ is not a member of ‘Genode::Board_base’
  {Genode::Board_base::I2C1_MMIO_BASE, Genode::Board_base::I2C_MMIO_SIZE},
   ^
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:91:39: error: ‘I2C_MMIO_SIZE’ is not a member of ‘Genode::Board_base’
  {Genode::Board_base::I2C1_MMIO_BASE, Genode::Board_base::I2C_MMIO_SIZE},
                                       ^
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:92:1: error: could not convert ‘{<expression error>, <expression error>}’ from ‘<brace-enclosed initializer list>’ to ‘I2C::Driver::I2C_bank’
 };
 ^
/home/krishna/tmp/genode-world.git/src/drivers/i2c/spec/zynq/driver.h:92:1: error: could not convert ‘{<expression error>, <expression error>}’ from ‘<brace-enclosed initializer list>’ to ‘I2C::Driver::I2C_bank’
```

**Comment by nfeske** [comment]
The problem with the SD-card driver might be related to the recent rework of the block layer by @skalk

**Comment by nfeske** [comment]
The i2c build issue is addressed by ef756be84048e9efd2ae24c9174b7e1d9290e290, which I just merged into master.

**Comment by nfeske** [comment]
The SD-card issue was not related to @skalk's work. It was the same problem as with the i2c driver. Both are fixed in master now.
